src/segments/cf_target.go

```diff
@@ -0,0 +1,79 @@
+package segments
+
+import (
+	"oh-my-posh/environment"
+	"oh-my-posh/properties"
+	"regexp"
+)
+
+type CfTarget struct {
+	props properties.Properties
+	env   environment.Environment
+
+	Target CfTargetDetails
+}
+
+type CfTargetDetails struct {
+	APIURL string
+	User   string
+	Org    string
+	Space  string
+}
+
+func (c *CfTarget) Template() string {
+	return "{{if .Target.Org }}{{ .Target.Org }}{{ end }}{{if .Target.Space }}/{{ .Target.Space }}{{ end }}"
+}
+
+func (c *CfTarget) Init(props properties.Properties, env environment.Environment) {
+	c.props = props
+	c.env = env
+}
+
+func (c *CfTarget) Enabled() bool {
+	c.setCFTargetStatus()
+
+	return c.Target != (CfTargetDetails{})
+}
+
+func (c *CfTarget) getCFTargetCommandOutput() string {
+	if !c.env.HasCommand("cf") {
+		return ""
+	}
+
+	output, err := c.env.RunCommand("cf", "target")
+
+	if err != nil {
+		return ""
+	}
+
+	return output
+}
+
+func (c *CfTarget) setCFTargetStatus() {
+	output := c.getCFTargetCommandOutput()
+
+	if output != "" {
+		regex := regexp.MustCompile(`API endpoint:\s*(?P<api_url>http[s].*)|user:\s*(?P<user>.*)|org:\s*(?P<org>.*)|space:\s*(?P<space>(.*))`)
+		match := regex.FindAllStringSubmatch(output, -1)
+		result := make(map[string]string)
+
+		for i, name := range regex.SubexpNames() {
+			if i != 0 && name != "" {
```

Comment:
we always use `len(name) != 0`

I would also invert the statement and continue rather than indent the entire block:

```go
if i == 0 && len(name) == 0 {
    continue
}
...
```

Reply:
Noted, will correct.

Reply:
Corrected.